packages/deployment/app/cloudformation.template.yml

```diff
@@ -425,7 +425,7 @@ Resources:
       Targets:
         - Id: {{@../key}}WatcherScheduler
           Input:
-            Fn::Sub: '{"queueUrl": "${ {{@../key}}SQS}", "messageLimit": {{this.messageLimit}} }'
+            Fn::Sub: '{"queueUrl": "${ {{@../key}}SQS}", "messageLimit": {{this.messageLimit}}, "timeLimit": 60 }'
```

Comment:
It's a time limit for the lambda execution. It runs every minute, and the current config is for it to run for 120 seconds, which causes overlapping executions. Those don't matter a _ton_, but I figured it would be cleaner to time-box visiblity and executions so we can guarantee no duplicate workflow starts.